subprojects/core/src/integTest/groovy/org/gradle/api/SettingsIncludeManyIntegrationTest.groovy

```diff
@@ -33,6 +33,9 @@ class SettingsIncludeManyIntegrationTest extends AbstractIntegrationSpec {
 
     @Requires(UnitTestPreconditions.IsGroovy3)
     def "including over 250 projects is not possible via varargs in Groovy 3"() {
+        createDirs(projectNames.take(254).collect({ name ->
```

Comment:
💅 I think it would be a bit clearer to extract some common logic for this and similar tests in a helper method.